packages/browser-data-collector/package.json

```
@@ -17,9 +18,9 @@
 		"url": "https://github.com/Automattic/wp-calypso/issues"
 	},
 	"scripts": {
-		"clean": "tsc --build ./tsconfig.json --clean && tsc --build ./tsconfig-cjs.json --clean",
-		"prepublish": "yarn run clean",
-		"prepare": "tsc --build ./tsconfig.json && tsc --build ./tsconfig-cjs.json",
+		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && npx rimraf dist",
+		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json",
```

I think this is a mistake:

```suggestion
		"build": "tsc --build ./tsconfig.json && tsc --build ./tsconfig-cjs.json",
```

---

I see this repeated several times, does `--build` work like is done here building both esm and cjs in our case?

---

`tsc --build [package1] [package2]` is a supported syntax. From https://www.typescriptlang.org/docs/handbook/project-references.html#tsc--b-commandline:

> `> tsc -b foo/prd.tsconfig.json bar  # Use foo/prd.tsconfig.json and bar/tsconfig.json`

---

That's great, thank you!